handbook/community.md

```diff
@@ -188,24 +189,78 @@ graph TD;
+- Ensure that checklist for submitter is complete.
+- Verify that all necessary reviews have been approved.
+- Merge the PR.
+- Thank and congratulate the contributor.
+- Share with the team in the #help-promote channel of Fleet Slack to be publicized on social media. Those who contribute to Fleet and are recognized for their contributions often become great champions for the project.
 
-### Process
+### Reviewing PRs from the community
 
-1. Decide if the change is acceptable (see below). If this will take time, acknowledge the contribution and let the user know that the team will respond. If you choose not to approve a change, first thank the contributor for their interest. Next, encourage them to open an issue or share their ideas before writing more code. Discussions can occur in the #fleet channel of osquery's Slack.
-2. Help the contributor make the content suitable for merging. Ensure appropriate manual and automated testing occurred. Also, confirm that changes to files and documentation are updated. Usually, this is best done with code review and coaching the user. Sometimes Fleet may complete a PR by improving testing and code review.
-3. After reviewing and addressing all the changes a PR needs, a Fleet team member may merge. Before merging, please use your best judgment. Double-check that CI is passing, documentation is updated, and a changes file is created.
-4. Once a PR is approved and merged, thank and congratulate the contributor. Next, share it with the team in the #help-promote channel of Fleet's Slack so they can promote it on social media. Recognized contributors often become project champions.
+If you're assigned a community pull request for review, it is important to keep things moving for the contributor. The goal is to not go more than one business day without following up with the contributor.
 
-Please refer to our [PRs from the community](https://docs.google.com/document/d/13r0vEhs9LOBdxWQWdZ8n5Ff9cyB3hQkTjI5OhcrHjVo/edit?usp=sharing) guide for more detailed information.
+A PR should be merged if:
+    - It's a change that is needed and useful. 
+    - The CI is passing.
+    - Tests are in place.
+    - Documentation is updated.
+    - Changes file is created.
 
-### What is acceptable?
+For PRs that aren't ready to merge:
+    - Thank the contributor for their hard work and explain why we aren't merging the changes.
+    - Encourage the contributor to reach out in the #fleet channel of osquery Slack to get help from the rest of the community.
+    - Offer code review and coaching to help get the PR ready to go.**
+    - Keep an eye out for any updates or responses.
 
-Everyone can contribute bug fixes and improvements to the documentation, whether or not they work for Fleet. Working Standard Query Library(SQL) additions or fixes are acceptable with correct attribution. Please use your best judgment.
+If everything is good to go, approve the review.
```

Comment:
Add a section: "For PRs that will not be merged"

Probably including something like

- Thank the contributor for their hard work and explain why we aren't merging the changes.
- Close the PR.